pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3456,6 +3494,29 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     int parent_tgid = get_task_host_tgid(parent);
     int child_tgid = get_task_host_tgid(child);
 
+    proc_info_t *c_proc_info = bpf_map_lookup_elem(&proc_info_map, &child_tgid);
+    if (c_proc_info == NULL) {
+        // this is a new process (and not just another thread) - add it to proc_info_map
+
+        proc_info_t *p_proc_info = bpf_map_lookup_elem(&proc_info_map, &parent_tgid);
+        if (unlikely(p_proc_info == NULL)) {
+            // parent proc should exist in proc_map (init_event_data should have set it otherwise)
+            // todo: log an error here
+            return 0;
+        }
+
+        bpf_map_update_elem(&proc_info_map, &child_tgid, p_proc_info, BPF_NOEXIST);
+        c_proc_info = bpf_map_lookup_elem(&proc_info_map, &child_tgid);
+        // appease the verifier
+        if (unlikely(c_proc_info == NULL)) {
+            // todo: log an error here
```

Comment:
@yanivagman we can already log. Have you tried?

Reply:
Done